packages/sdk/src/BoostCore.ts

```diff
@@ -1023,6 +1027,41 @@ export class BoostCore extends Deployable<
     return { hash, result };
   }
 
+  /**
+   * Retrieves the claim information from a transaction receipt.
+   *
+   * @param {GetTransactionReceiptParameters} params - The parameters required to get the transaction receipt.
+   * @returns {Promise<{ boostId: bigint, incentiveId: bigint, claimer: Address, amount: bigint } | undefined>} The claim information if found, undefined otherwise.
+   *
+   * @description
+   * This method retrieves the transaction receipt using the provided parameters,
+   * then parses the logs to find the 'BoostClaimed' event.
+   * If found, it returns the arguments of the event, which include the boost ID,
+   * incentive ID, claimer address, and claimed amount.
+   *
+   * @example
+   * ```ts
+   * const claimInfo = await boostCore.getClaimFromTransaction({
+   *   hash: '0x...',
+   *   chainId: 1
+   * });
+   * if (claimInfo) {
+   *   console.log(`Boost ${claimInfo.boostId} claimed by ${claimInfo.claimer}`);
+   * }
+   * ```
+   */
+  public async getClaimFromTransaction(
+    params: GetTransactionReceiptParameters,
+  ) {
+    const receipt = await getTransactionReceipt(this._config, params);
+    const logs = parseEventLogs({
+      abi: boostCoreAbi,
+      eventName: 'BoostClaimed',
+      logs: receipt.logs,
+    });
+    return logs.at(0)?.args;
```

Comment:
I don't think `getTransactionReceipt` or `parseEventLogs` return `args`

I think we may need to decode manually here?
https://viem.sh/docs/contract/decodeEventLog

Reply:
Might also be able to pass `strict: false` in `parseEventLogs`, but I haven't tried that before
https://viem.sh/docs/contract/parseEventLogs#partial-decode